lib/src/http_body.dart

```diff
@@ -222,24 +222,24 @@ Future<HttpBody> _process(Stream<List<int>> stream, HttpHeaders headers,
 
   Future<HttpBody> asBinary() async {
     var builder = await stream.fold(
```

Comment:
```suggestion
    var builder = await stream.fold<BytesBuilder>(
```

And then remove `dynamic` below